rj_gameplay/rj_gameplay/tactic/pass_tactic.py

```diff
@@ -222,18 +283,27 @@ def tick(self, world_state: rc.WorldState,
         """
         pivot_result = role_results[self.pivot_kick]
         receive_result = role_results[self.receive]
-        if receive_result and receive_result[0].is_filled():
-            self.passer_cost.chosen_receiver = receive_result[0].role.robot
-            self.pivot_kick.skill.target_point = np.array(receive_result[0].role.robot.pose[0:2])
-            self.passer_cost.potential_receiver = receive_result[0].role.robot
-            return [self.receive]
-        elif pivot_result and pivot_result[0].is_filled():
-            potential_receiver = self.find_potential_receiver(world_state)
-            self.pivot_kick.skill.target_point = np.array(
-                [potential_receiver.pose[0], potential_receiver.pose[1]])
+        if pivot_result and pivot_result[0].is_filled():
+            self.receiver_cost.passer_robot = pivot_result[0].role.robot
+            # self.pivot_kick.skill.target_point = np.array(receive_result[0].role.robot.pose[0:2])
+            self.pivot_kick.skill.target_point = self.find_potential_receiver(world_state).pose[0:2]
+            self.pivot_kick.skill.pivot_point = np.array(
+                pivot_result[0].role.robot.pose[0:2])
+
+            # self.receiver_cost.potential_receiver = receive_result[
```

Comment:
delete this commented out code if it's not being used, or leave comment explaining why it should stay